tests/sentry/api/endpoints/test_project_details.py

```diff
@@ -378,193 +365,180 @@ def test_options(self):
+            self.proj_slug,
+            allowedDomains=["foobar.com", "https://example.com"],
         )
-        assert resp.status_code == 200, resp.content
         assert self.project.get_option("sentry:origins") == ["foobar.com", "https://example.com"]
         assert resp.data["allowedDomains"] == ["foobar.com", "https://example.com"]
 
         # cannot be empty
-        resp = self.client.put(self.path, data={"allowedDomains": ""})
-        assert resp.status_code == 400, resp.content
+        resp = self.get_valid_response(
+            self.org_slug, self.proj_slug, allowedDomains="", status_code=400
+        )
         assert self.project.get_option("sentry:origins") == ["foobar.com", "https://example.com"]
         assert resp.data["allowedDomains"] == [
             "Empty value will block all requests, use * to accept from all domains"
         ]
 
-        resp = self.client.put(self.path, data={"allowedDomains": ["*", ""]})
-        assert resp.status_code == 200, resp.content
+        resp = self.get_valid_response(
+            self.org_slug,
+            self.proj_slug,
+            allowedDomains=["*", ""],
+        )
         assert self.project.get_option("sentry:origins") == ["*"]
         assert resp.data["allowedDomains"] == ["*"]
 
     def test_safe_fields(self):
-        resp = self.client.put(
-            self.path, data={"safeFields": ["foobar.com", "https://example.com"]}
+        resp = self.get_valid_response(
+            self.org_slug,
+            self.proj_slug,
+            safeFields=["foobar.com", "https://example.com"],
         )
-        assert resp.status_code == 200, resp.content
         assert self.project.get_option("sentry:safe_fields") == [
             "foobar.com",
             "https://example.com",
         ]
         assert resp.data["safeFields"] == ["foobar.com", "https://example.com"]
 
     def test_store_crash_reports(self):
-        resp = self.client.put(self.path, data={"storeCrashReports": 10})
-        assert resp.status_code == 200, resp.content
+        resp = self.get_valid_response(self.org_slug, self.proj_slug, storeCrashReports=10)
         assert self.project.get_option("sentry:store_crash_reports") == 10
         assert resp.data["storeCrashReports"] == 10
 
     def test_relay_pii_config(self):
         value = '{"applications": {"freeform": []}}'
-        resp = self.client.put(self.path, data={"relayPiiConfig": value})
-        assert resp.status_code == 200, resp.content
+        resp = self.get_valid_response(self.org_slug, self.proj_slug, relayPiiConfig=value)
         assert self.project.get_option("sentry:relay_pii_config") == value
         assert resp.data["relayPiiConfig"] == value
 
     def test_sensitive_fields(self):
-        resp = self.client.put(
-            self.path, data={"sensitiveFields": ["foobar.com", "https://example.com"]}
-        )
-        assert resp.status_code == 200, resp.content
+        value = ["foobar.com", "https://example.com"]
```

Comment:
not a big deal but why did you assign this as a var when in other places (like line 473) you don't? I think consistency is good